test/README.md

```diff
@@ -0,0 +1,182 @@
+# Test
+This directory contains tests and testing docs for the knative-sandbox `eventing-kafka` project.
+
+- [Unit Tests](#unit-tests) reside in the codebase alongside the code under test.
+- [Integration Tests](#integration-tests) consist of the `conformance` and `e2e` tests also in this directory.
+- [Performance Tests](#performance-tests) have yet to be completed but will reside here as well.
+
+While some information has been repeated here, it is useful to be familiar with the other
+Knative testing documentation such as...
+- [Serving](https://github.com/knative/serving/blob/master/test/README.md)
+- [Eventing](https://github.com/knative/eventing/tree/master/test/README.md)
+- [Eventing-Contrib](https://github.com/knative/eventing-contrib/blob/master/test/README.md) 
+
+
+## Unit Tests
+The Unit Tests are run by Prow for all Pull Requests.  They are executed by Prow via the [
+pre-submit-tests.sh](./presubmit-tests.sh) script with the `--unit-tests` argument. 
+
+The project [Makefile](../Makefile) contains convenience targets for executing the unit-tests
+for various portions of the project with a code coverage summary...
```

Comment:
Need to fix the Makefile path for new structure and/or rework language to indicate the various components?